packages/design-system-web/src/components/geov-entity-props-by-predicate/geov-entity-props-by-predicate.css

```diff
@@ -12,3 +12,21 @@ ion-card {
 geov-paginator {
   margin-left: auto;
 }
+
+.literal-container {
+  overflow: clip;
+  width: 100%;
+}
+
+.literal-expanded .sc-ion-label-ios-h,
+.literal-expanded .sc-ion-label-ios-s {
+  white-space: normal;
+}
+
+.text-right ion-button {
+  margin-left: auto;
+}
+
+.literal-container .item-button {
+  display: none;
+}
```

Comment:
see my comment about radically change the approach from collapse/expand to modal

Reply:
Done. Only the expanded/collapsed classes are removed. Classes related to the button remain